starter_code/car.js

```diff
@@ -0,0 +1,29 @@
+function CreateCarObject() {
+    this.x = 220;
+    this.speed = 10;
+    this.img = new Image();
+    this.scale = 158 / 319;
+}
+
+CreateCarObject.prototype.move = function(key){
+    switch (key) {
+        case 37: // left
+          this.x -= 5;
+          if (this.x < 20) {
+            this.x = 20;
+          }
+          break;
+        case 39: // right
+          this.x += 5;
+          if (this.x > 265) {
+            this.x = 265;
+          }
+          break;
+        default:
+          return; // exit this handler for other keys
+      }
+  }
+
+CreateCarObject.prototype.render = function(ctx){
```

Comment:
Intenta el dibujado del propio coche esté en su método, con el image.onload tb, por ejemplo:

CreateCarObject.prototype._AddCar = function (canvas) {
  var img = new Image();
  img.src = './images/car.png';
  img.onload = function() {
  canvas.ctx.drawImage(img,this.x,this.y,40,80);
}.bind(this);
};